openedx/core/djangoapps/user_api/accounts/toggles.py

```diff
@@ -25,20 +25,19 @@ def should_redirect_to_order_history_microfrontend():
     )
 
 
-# .. toggle_name: account.redirect_to_microfrontend
-# .. toggle_implementation: WaffleFlag
+# .. toggle_name: FEATURES['ENABLE_ACCOUNT_MICROFRONTEND']
+# .. toggle_implementation: SettingDictToggle
 # .. toggle_default: False
-# .. toggle_description: Supports staged rollout of a new micro-frontend-based implementation of the account page.
-# .. toggle_use_cases: temporary, open_edx
```

Comment:
A `temporary` toggle means that we don't intend to support the toggle indefinitely. So, I think the right move would be to delete this toggle altogether, including the code behind the non-MFE case, but only once it is ready. Also, the target removal data could be adjusted if this is a more realistic date.

I don't see any benefit to simply adjusting the toggle type.

Reply:
@robrap this change allows to enable/disable the MFE globally. Considering a multisite installation, the MFE could be enabled for all sites by just setting this Toggle to True (no need to create all Site configuration entries).

Reply:
@jfavellar90: Thanks for bringing to my attention the other intent of this PR. I presume the main change for making this global is the following:
```
configuration_helpers.get_value('ENABLE_ACCOUNT_MICROFRONTEND') and
        REDIRECT_TO_ACCOUNT_MICROFRONTEND.is_enabled()
```
to
```
configuration_helpers.get_value('ENABLE_ACCOUNT_MICROFRONTEND', ENABLE_ACCOUNT_MICROFRONTEND.is_enabled())
```
Is that correct?

And another way to accomplish this would be to use:
```
configuration_helpers.get_value('ENABLE_ACCOUNT_MICROFRONTEND',
        REDIRECT_TO_ACCOUNT_MICROFRONTEND.is_enabled())
```
?

1. The toggle type and name change introduces some consistency of naming, as noted in the PR description, but it also introduces backward incompatibility. This may be fine (not my call), but I'd use `refactor!` for your commit title and add a much longer commit description for operators that are already using this MFE who will need to add the new setting, and would need to clean-up the existing waffle flag. Without this, this change would disable the MFE, right? Note: The original commit message has the typo `Toogle`.
2. This refactored toggle, if we continue with the refactor, should still be `temporary` and still have all the other annotations. If you see DEPR-17, the ultimate goal is to have the MFE be the only option. Once the MFE is the only option, this toggle will no longer be needed.

Reply:
```
configuration_helpers.get_value('ENABLE_ACCOUNT_MICROFRONTEND',
        REDIRECT_TO_ACCOUNT_MICROFRONTEND.is_enabled())
```

This approach suggested by @robrap is more light-touched and less backward-compatible. It enables the account microfrontend by default, if `REDIRECT_TO_ACCOUNT_MICROFRONTEND` is enabled, but it also makes it possible to disable the MFE per site.

Still, I think it's worthwhile to examine the possibility of getting rid of the site configuration flag: the less site configuration flags we have, the better.

Reply:
@jfavellar90 
1. I'm more ok with the other backward-incompatible change that @regisb proposed, to remove the check for the site configuration flag. I don't think we are using it. However, it does add additional risk with not much benefit since the entire flag and option will be going away. Also, I don't know if this would need a separate DEPR ticket to inform it is going away early?
2. I'm not a fan of needlessly changing the toggle type (a backward-incompatible change) for a toggle that is supposed to be removed anyway. It just gives us more work to ensure nothing gets broken.
3. In all cases, please restore the `temporary` use case and other annotations.

If you feel strongly about changing the toggle type and name, I think @kashifch would need to weigh-in about whether his team wants to support this change.

Thank you!

Reply:
@robrap I'm not sure I understand: are you agreeing with me that we should remove the site configuration flag? If yes, we would still have to gate the feature with a toggle -- and I think that a SettingDictToggle would be the best solution. But that will effectively result in a toggle type change, which you say is a bad thing?